docs/plugins/components.mdx

```diff
@@ -0,0 +1,856 @@
+---
```

Comment:
Can you update the ESLint config to support mdx? Just replace `md` with `{md,mdx}`.